CHARTER.md

```diff
@@ -0,0 +1,73 @@
+
+
+Technical Charter (the “Charter”)
+for 
+Besu a Series of LF Projects, LLC
+
+Adopted ___________
+
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Besu open source project, which has been established as Besu a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+1.	Mission and Scope of the Project
+    a.	The mission of the Project is to develop an Ethereum client designed to be enterprise and infrastructure-friendly for both public and private permissioned network use cases, with an extractable EVM implementation.
+    b.	The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+2.	Technical Steering Committee
+    a.	The Technical Steering Committee (the “TSC”) will be responsible for all technical oversight of the open source Project. 
+    b.	The TSC voting members are initially the Project’s Committers. At the inception of the project, the Committers of the Project will be as set forth within the “CONTRIBUTING” file within the Project’s code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
```

Comment:
our  https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md file does not say anything about committers or TSC